diff -ruN .cache/node/globals.d.ts types/node/globals.d.ts
--- .cache/node/globals.d.ts	2025-06-05 10:36:57.222480830 +0000
+++ types/node/globals.d.ts	2025-06-05 10:36:56.122481685 +0000
@@ -313,6 +313,16 @@
     // #region fetch
     interface RequestInit extends _RequestInit {}
 
+    /**
+     * @throws {Promise<DOMException>} AbortError if the request is aborted.
+     */
+    function fetch(
+        input: string | URL | globalThis.Request,
+        init: { signal: AbortSignal } & RequestInit,
+    ): Promise<Response>;
+    /**
+     * https://nodejs.org/docs/latest/api/globals.html#fetch
+     */
     function fetch(
         input: string | URL | globalThis.Request,
         init?: RequestInit,
